.eslintrc.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
     // Problems with optional parameters
     '@typescript-eslint/no-unnecessary-condition': 'off',
     '@typescript-eslint/prefer-optional-chain': 'error',
+    '@typescript-eslint/promise-function-async': [ 'error', { checkArrowFunctions: false } ],
```

Comment:
Why was this disabled? I don't think it's bad to have this enabled.

Reply:
For the case below; in arrow functions like these, where we are just returning promise, it's unnecessary overhead:

```
(handler): Promise<void> => handler.canHandle(input))
```

so I think disabling it for arrows makes sense. But up to you.